Vizzuality/trase
# Feature/top sources of import chart

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/172552982

## Description

Importer changes were necessary to pull import data per partner (rather than world aggregates). The job that updates that data from the ComTrade API takes a long time, because of the need to throttle requests in order not to exceed free usage limits.

## Testing instructions

The ComTrade job now runs on the first Friday every month; in order to force start a run, comment out the day of month check in the importer and start it in the console: 

```
importer = Api::V3::CountriesComTradeIndicators::ImporterService.new
importer.call
```

To update configuration of the charts:

`bundle exec rake db:country_profiles:clear_and_populate`
